applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_3d.py

```diff
@@ -361,7 +361,7 @@ def __VisualizeWake(self):
             elem.Id = counter
             counter +=1
 
-        from gid_output_process import GiDOutputProcess
+        from KratosMultiphysics.gid_output_process import GiDOutputProcess
```

Comment:
Please move all the imports to the top part of the file. We experienced some issues with recursive imports in the past. Besides this is a good practise according to the PEP-8 style guide.

Reply:
Ok!